test/server-unit/xlsx.js

```diff
@@ -0,0 +1,33 @@
+const xlsx = require('../../server/lib/renderers/xlsx');
+
+const { expect } = require('chai');
+const _ = require('lodash');
+
+describe('util.js', () => {
+
+  it('Shoul return a xlsx buffer', (done) => {
+    const data = {
+      rows : [{ Firstname : 'Alice', Lastname : 'Bob' }],
+    };
+    xlsx.render(data)
```

Comment:
There isn't any need to use `done()` in these callbacks.  It works, but promises are prettier.  All you need to do is return the `xlsx.render()` call.  Like this:
```js
it('Should return a xlsx buffer', () => {
  const data = { /* some data */ };
  return xlsx.render(data).then((reportStream) => {
     expect(_.isBuffer(reportStream)).to.be.equal(true);
   });
});
```

Reply:
[This is a good article](http://tobyho.com/2015/12/16/mocha-with-promises/) about it.